crates/store/re_video/src/demux/mod.rs

```diff
@@ -89,6 +89,11 @@ pub struct SamplesStatistics {
     ///
     /// If true, the video typically has no B-frames as those require frame reordering.
     pub dts_always_equal_pts: bool,
+
+    /// If `dts_always_equal_pts` is false, then this gives for each sample whether its PTS is the highest seen so far.
+    /// If `dts_always_equal_pts` is true, then this is left empty.
+    /// This is used for optimizing PTS search.
+    pub has_sample_highest_pts_so_far: Vec<bool>,
```

Comment:
Would it make sense to use `bit-vec` instead here?

Reply:
ah, yes good call!